src/com/google/enterprise/adaptor/filenet/DocumentTraverser.java

```diff
@@ -213,20 +212,23 @@ private void processDocument(Id guid, DocId docId, Document document,
     String vsDocId = document.get_VersionSeries().get_Id().toString();
     logger.log(Level.FINE, "VersionSeriesID for document is: {0}", vsDocId);
 
-    ActiveMarkingList activeMarkings = document.get_ActiveMarkings();
-    Permissions.Acl permissions =
-        new Permissions(document.get_Permissions(), document.get_Owner())
-        .getAcl();
-    response.setAcl(getAcl(docId, permissions));
-    processInheritedPermissions(docId, activeMarkings, permissions, response);
+    if (options.markAllDocsAsPublic()) {
+      response.setSecure(false);
```

Comment:
That's redundant. I'm not really sure why I added the call to setSecure. None of the other connectors use it.